db/import.js

```diff
@@ -0,0 +1,62 @@
+import dotenv from "dotenv";
+dotenv.config({ path: ".env.local" });
+import connect, { disconnect } from "./connect.js";
+import { users, starships } from "./fixtures.js";
+import User from "./models/User.js";
+import Starship from "./models/Starship.js";
+import Listing from "./models/Listing.js";
+
+function random(max, min) {
+  return Math.floor(Math.random() * (max - min) + min);
+}
+
+// sampleSize
+function getStarshipId(arr) {
+  return arr[random(arr.length - 1, 0)]._id;
+}
+
+console.log("Connecting...");
+await connect(process.env.MONGODB_URI);
+
+console.log("Dropping existing collections...");
+await Listing.collection.drop();
+await Starship.collection.drop();
+await User.collection.drop();
+
+const createdStarships = await Starship.insertMany(starships);
+
+await Promise.all(
+  users.map(async ({ listings, ...user }, index) => {
+    console.log(`Creating user: ${index}...`);
```

Comment:
Just for clean code reasons, you could refactor the code and delete the console.log later.